content/en/docs/architecture/bugzilla.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Bugzilla Integration"
+description: An overview of the integration of Bugzilla with the Openshift CI
+---
+
+**Note:** This documentation describes how to configure CI to assist with Bugzilla-related processes. It does _not_ describe
+how these processes look. Documentation about the Openshift Bugzilla process can be found [here][6].
+
+## Linking Pull Requests to Bugzilla bugs
+
+Pull requests can reference a Bugzilla bug in their title, for example
+`Bug 123: Rest of the PR title`. A PR can at most reference one Bugzilla bug. Referencing a bug makes the bugzilla plugin run a couple
+of checks:
+* If a bug with the id `123` exists
```

Comment:
I think for the general overview, we should let the plugin do it's thing and not over-explain here. Think about - for any of these validations - is the user who's fixing the bug responsible for _doing anything_ themselves? If they've been assigned a bug to work on, I don't think they have to do anything necessarily to make it valid, and if so the plugin will let them know. So, from an instructional point-of-view when we're describing what the user has to be doing and thinking about, we don't need this level of detail.

Reply:
What sorts of things does the plugin not explain in-PR?  [Here][1] is the bot explaining that I need a `Bug ###` reference.  Once I add one, it will comment explaining the verification checks and which, if any, have failed.  How many help-desk threads are "I am not reading those in-PR comments", how many are "the in-PR comments are not answering my questions", and how many are "the in-PR comments could not possibly answer my questions"?  It's not clear to me if anything falls into that third category, and it's not clear to me that the first category can be resolved by adding more docs further away from the PR at issue.

[1]: https://github.com/openshift/cluster-version-operator/pull/536#issuecomment-806003890

Reply:
I agree, @wking. That's why I think what we need here is a minimal "in order to fix a bug with your PR, add the prefix to your title" message and let the plugin run its course after that.